RFC/src/RFC-0201_TariScript.md

```diff
@@ -360,7 +374,33 @@ pub struct Transaction {
 }
 ```
 
-And Blockheaders need to be modified to:
+All _script offsets_ (\\(\so\\)) from (6) contained in a block is summed together to create a _total script offset_ (8) 
+so that algorithm (6) still holds for a block.
+
+$$
+\begin{aligned}
+\so_{total} = \sum_k\mathrm{\so_{k}}\\; \text{for every transaction}, k
+ \end{aligned}
+ \tag{8}
+$$
+
+Verification of (8) will entail:
+
+$$
+\begin{aligned}
+\so_{total} \cdot G = \sum_j\mathrm{K_{Sj}} - \sum_i\mathrm{K_{Oi}} \\; \text{for each input}, j,\\, \text{and each output}, i
+ \end{aligned}
+ \tag{9}
+$$
+
+As can be seen all information required to verify (8) is contained in a block's inputs and outputs. One important 
+distinction to make is that the Coinbase UTXO does not count towards the _script offset_. This is because the Coinbase 
+UTXO already has special rules accompanying it and it has no input, thus we cannot generate a _script offset_ 
+\\( \so \\) for a Coinbase transaction.
+The coinbase can allow any _script_ \\(\script_i\\) and  _script offset public key_ \\( K\_{Oi} \\) as long as it does 
```

Comment:
why not make the script offset pub key and signature empty for the coinbases? Seems like a waste to check this signature

Reply:
Coinbase script and script_offset_keys are not checked when creating the coinbase output. But this allows a user to create a custom script for his coinbase that still needs validating on spending. This allows for example the miner to lock up the coinbase for children based on certain criteria. The meta_signature should still be singed to prevent malleability on the metadata such as the script.

Reply:
But pub key used to sign the script is also malleable, so the attacker would just change that as well